02-peer-protocol.md

```diff
@@ -1033,6 +1036,21 @@ A sending node:
   - MUST increase the value of `id` by 1 for each successive offer.
   - if it is relaying a payment inside a blinded route:
     - MUST set `blinding_point` (see [Route Blinding](04-onion-routing.md#route-blinding))
+  - if it is the original source of the HTLC:
+    - if it does not expect immediate fulfillment upon receipt by the 
+      final destination: 
+      - SHOULD set `endorsed` to `0`.
+    - otherwise:  
+      - SHOULD set `endorsed` to `1`. 
```

Comment:
OK maybe I am missing something but when and how the node will choose when set the value to `1` or to `0` in the `otherwise` case?

Reply:
If the node sending the payment is not expecting delays, they should set `endorsed` to `1`, this is the `otherwise` case.
That is, the node should set `endorsed` to `0` only if it expects the payment to resolve slowly (because of a non-standard use case, for example).

Reply:
>If the node sending the payment is not expecting delays, they should set endorsed to 1,

I found this description much more clear, thanks 😸